dotnet-interactive/dotnet-interactive.csproj

```diff
@@ -126,14 +126,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <TypescriptInputs Include="$(JsApiRoot)mocha.opts" />
```

Comment:
If changes to this file can affect the resulting .js files, then it should remain in the item group.  If not, e.g., test only, then the removal is fine.

Reply:
And to add a clarifying point for @jonsequitur, inclusion in the `@(TypescriptInputs)` group does not mean the file will end up in the package, it simply means that changes to any of these files invalidate the output of the TypeScript compilation.

Reply:
that file contains settings for running tests only